.github/workflows/ci-actions.yml

```diff
@@ -206,17 +206,17 @@ jobs:
           name: test-reports-windows-jdk11-jvm
           path: 'test-reports.tgz'
 
-  linux-jvm-gradle-tests:
-    name: Gradle Tests JDK ${{matrix.java.name}}
+  linux-jvm-maven-tests:
+    name: Maven Tests - JDK ${{matrix.java.name}}
     runs-on: ubuntu-latest
     needs: build-jdk11
     timeout-minutes: 60
     strategy:
       fail-fast: false
       matrix:
         java:
-         - {
-            name: Java 11,
+          - {
```

Comment:
Hopefully this whitespace doesn't break anything :)

Reply:
The others were indented with 2 spaces so I made it consistent.